### PR TITLE
Asc io file driver

### PR DESCRIPTION
**Pull request by nicolas-f**
Add Arc Ascii file reader
- No dependencies
- Default to import pixels as polygons with attribute Z as a column (NODATA set null as Z field)
- Option to import pixels as Point (X, Y, Z) at center of pixel (NODATA skip the pixels)
- Option to import only pixels following a provided Envelope
- Option to reduce the number of pixels imported (downscaled, for a quick preview)

### CODE REVIEW

File: h2gis-functions/src/main/java/org/h2gis/functions/io/asc/AscDriverFunction.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Asc driver to import ESRI ASCII Raster file as polygons
+ * 
+ * @author Nicolas Fortin (Université Gustave Eiffel 2020)
```
Comment (ebocher) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/asc/AscDriverFunction.java]: Université Gustave Eiffel 2020, UMRAE ?

Comment (nicolas-f) [reply h2gis-functions/src/main/java/org/h2gis/functions/io/asc/AscDriverFunction.java]: Bah ouai, t'est pas au courant ?

**Comment by nicolas-f** [comment]
Vous voulez que j'ajoute AscDriverFunction et ASCRead dans des tests unitaires ?

**Comment by nicolas-f** [comment]
J'ajoute les tests pour ASCReader

**Comment by nicolas-f** [comment]
Don't know what to do with NPath complexity..

**Comment by ebocher** [comment]
In many GIS the default behaviour is to import ascii file as points not polygon

**Pull request merged**